tests/integration/test_design_import.py

```diff
@@ -144,7 +144,6 @@ def test_design_import_with_surfaces_issue834(modeler: Modeler):
         assert design.bodies[1].is_surface == True
 
 
-@pytest.mark.skip(reason="Get the OpenSSL GeometryService through before fixing hoops")
 def test_open_file(modeler: Modeler, tmp_path_factory: pytest.TempPathFactory):
     """Test creation of a component, saving it to a file, and loading it again to a
     second component and make sure they have the same properties."""
```

Comment:
```suggestion
    """Test creation of a component.

    This method saves the open file to a new file. It, then loads the new
    file to a second component to make sure that they have the same
    properties."""
```
I hope I understood what was there. If not, I'm sure you understand the need for improving what was originally there.